Affiliated-Project-Policy.md

```diff
@@ -127,11 +127,11 @@ The status of Affiliated and Candidate OBF Projects that are not voted on stays
 
   Affiliated OBF projects that have not been actively maintained for over twelve months may have their affiliation status archived as a result of a review by the OBF Board of Directors. The OBF Board of Directors will conduct such reviews in regular intervals, and will make reasonable efforts to contact the project maintainers before taking action.
 
-## Projects with grandfathered affiliation status
+## Projects with 'Legacied' Affiliation Status
```

Comment:
Use pre-existing here?

Reply:
```suggestion
## Pre-existing OBF Projects Granted Affiliated Status
```

Reply:
Replaced Legacied with Granted in all sentences

Reply:
I prefer this section title "Pre-existing OBF Projects Granted Affiliated Status".